src/Atmos/Model/turbulence.jl

```diff
@@ -1,10 +1,32 @@
-#### Turbulence closures
+# ## Turbulence Closures
+# In `turbulence.jl` we specify turbulence closures. Currently, 
+# pointwise models of the eddy viscosity/eddy diffusivity type are
+# supported for turbulent shear and tracer diffusivity. Methods currently supported
+# are:\
+# [`ConstantViscosityWithDivergence`](@ref constant-viscosity)\
+# [`SmagorinskyLilly`](@ref smagorinsky-lilly)\
+# [`Vreman`](@ref vreman)\
+# [`AnisoMinDiss`](@ref aniso-min-diss)\
+
+#md # !!! note
+#md #     Usage: This is a quick-ref guide to using turbulence models as a subcomponent
+#md #     of `AtmosModel` \
+#md #     $\nu$ is the kinematic viscosity, $C_smag$ is the Smagorinsky Model coefficient,
+#md #     `turbulence=ConstantViscosityWithDivergence(ν)`\
+#md #     `turbulence=SmagorinskyLilly(C_smag)`\
+#md #     `turbulence=Vreman(C_smag)`\
+#md #     `turbulence=AnisoMinDiss(C_poincare)`
+
 using DocStringExtensions
-using CLIMA.PlanetParameters: grav
-using CLIMA.SubgridScaleParameters
+using ..PlanetParameters: grav
+using CLIMA.SubgridScaleParameters: inv_Pr_turb
```

Comment:
@charleskawczynski This needs to be updated following `https://github.com/climate-machine/CLIMAParameters.jl/pull/10` right /

Reply:
`CLIMA.SubgridScaleParameters` is supported in the interim; while the CLIMAParameters repo is set up.